### ISSUE TITLE
Files not shown in assessment mode

### ISSUE THREAD

**Issue by luis100** [keeps/roda#863]
In assessment mode, when AIP is *not* active, files are not shown in the file explorer.

![files_not_shown_in_assessment_mode](https://cloud.githubusercontent.com/assets/631728/25099767/8741a6c0-23a5-11e7-8257-5e6d74bc6b57.png)

List should set the justActive flag to true *only if* AIP state is `ACTIVE`.

**Comment by luis100** [comment]
To note, the directory in questions has the following files:
```
➜  storage tree aip/0ab780fa-997e-45ce-970f-ced325f95243/representations/rep1/data/doc
aip/0ab780fa-997e-45ce-970f-ced325f95243/representations/rep1/data/doc
├── cas.md
└── images
    ├── kitematic_open.png
    └── kitematic_search.png

1 directory, 3 files
```

**Comment by luis100** [comment]
The example SIP for testing:
[sip.zip](https://github.com/keeps/roda/files/926367/sip.zip)

**Comment by jmaferreira** [comment]
This fix should have a high priority!

**Comment by nunovieira220** [comment]
Solved on c7676f58558e88c33604374e7038099f80ecc399
